### PR TITLE
Solve the timeout issue of digiCert Global Root CA download

### PR DESCRIPTION
**Pull request by Aymalla**
I've removed the download of the digiCert Global Root CA certificate because the server is now stable and causing timeout issues on the pipeline

### CODE REVIEW

File: .devcontainer.json [hunk 1]
```diff
@@ -31,7 +31,6 @@
     "ghcr.io/devcontainers/features/powershell:1": {
       "modules": "Az.Resources, Microsoft.Graph.Authentication, Microsoft.Graph.Applications"
     },
-    "ghcr.io/rchaganti/vsc-devcontainer-features/azurebicep:1": {
-    }
+    "ghcr.io/rchaganti/vsc-devcontainer-features/azurebicep:1": {}
```
Comment (ross-p-smith) [inline .devcontainer.json:34]: Is this the linter making this change? I just don't want someone else's settings reverting this back again! :-) Feel free to resolve this comment if it's the linter!

File: data-reconciliation-app/test/utils/jwt-config-generator.ts [hunk 2]
```diff
@@ -157,6 +150,18 @@ export class JwtConfigsGenerator {
...
...3cuZGlnaWNlcnQuY29tMSAwHgYDVQQDExdEaWdpQ2VydCBHbG9iYWwgUm9vdCBD\nQTAeFw0wNjExMTAwMDAwMDBaFw0zMTExMTAwMDAwMDBaMGExCzAJBgNVBAYTAlVT\nMRUwEwYDVQQKEwxEaWdpQ2VydCBJbmMxGTAXBgNVBAsTEHd3dy5kaWdpY2VydC5j\nb20xIDAeBgNVBAMTF0RpZ2lDZXJ0IEdsb2JhbCBSb290IENBMIIBIjANBgkqhkiG\n9w0BAQEFAAOCAQ8AMIIBCgKCAQEA4jvhEXLeqKTTo1eqUKKPC3eQyaKl7hLOllsB\nCSDMAZOnTjC3U/dDxGkAV53ijSLdhwZAAIEJzs4bg7/fzTtxRuLWZscFs3YnFo97\nnh6Vfe63SKMI2tavegw5BmV/Sl0fvBf4q77uKNd0f3p4mVmFaG5cIzJLv07A6Fpt\n43C/dxC//AH2hdmoRBBYMql1GNXRor5H4idq9Joz+EkIYIvUX7Q6hL+hqkpMfT7P\nT19sdl6gSzeRntwi5m3OFBqOasv+zbMUZBfHWymeMr/y7vrTC0LUq7dBMtoM1O/4\ngdW7jVg/tRvoSSiicNoxBN33shbyTApOB6jtSj1etX+jkMOvJwIDAQABo2MwYTAO\nBgNVHQ8BAf8EBAMCAYYwDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQUA95QNVbR\nTLtm8KPiGxvDl7I90VUwHwYDVR0jBBgwFoAUA95QNVbRTLtm8KPiGxvDl7I90VUw\nDQYJKoZIhvcNAQEFBQADggEBAMucN6pIExIK+t1EnE9SsPTfrgT1eXkIoyQY/Esr\nhMAtudXH/vTBH1jLuG2cenTnmCmrEbXjcKChzUyImZOMkXDiqw8cvpOp/2PV5Adg\n06O/nVsJ8dWO41P0jmP6P6fbtGbfYmbW0W5BjfIttep3Sp+dWOIrWcBAI+0tKIJF\nPnlUkiaY4IBIqDfv8NZ5YBberOgOzW6sRBc4L0na4UU+Krk2U886UAb3LujEV0ls\nYSEY1QSteDwsOoBrp+uvFRTp2InBuThs4pFsiv9kuXclVzDAGySj4dzp30d8tbQk\nCAUw7C29C79Fv1C5qfPrmAESrciIxpg0X40KPMbp1ZWVbd4=\n-----END CERTIFICATE-----\n`;
```
Comment (ross-p-smith) [inline data-reconciliation-app/test/utils/jwt-config-generator.ts:163]: Can we add a comment that states that the cert is hosted on an unstable server and this is why we don't download it

**Comment by ross-p-smith** [comment]
@achamayou - are you OK that we hardcode this Root Cert in the repo as we are getting timeouts occasionally downloading it

File: .devcontainer.json (hunk 1 above)
Comment (Aymalla) [reply .devcontainer.json:34]: Yes, it is linter. 
eslint and prettier are now part of package.json

File: data-reconciliation-app/test/utils/jwt-config-generator.ts (hunk 2 above)
Comment (Aymalla) [reply data-reconciliation-app/test/utils/jwt-config-generator.ts:168]: Done

**Review by achamayou — APPROVED** [review]

**Pull request merged**